webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerTitle.js

```diff
@@ -2,9 +2,13 @@ import React from 'react';
 
 const NotificationDrawerTitle = ({text}) => (
   <div className="drawer-pf-title">
-    <a className="drawer-pf-toggle-expand"></a>
```

Comment:
this probably best to do in a separate pr, as there is a redmine issue for it already.